Client/mods/deathmatch/logic/luadefs/CLuaWorldDefs.h

```diff
@@ -112,4 +116,8 @@ class CLuaWorldDefs : public CLuaDefs
     LUA_DECLARE(GetFPSLimit);
 
     LUA_DECLARE(CreateExplosion);
+
+    static bool ResetColorFilter(lua_State* const luaVM);
+    static bool SetColorFilter(lua_State* const luaVM, uchar ucPass0Red, uchar ucPass0Green, uchar ucPass0Blue, uchar ucPass0Alpha,
+        uchar ucPass1Red, uchar ucPass1Green, uchar ucPass1Blue, uchar ucPass1Alpha);
```

Comment:
Since the implementations aren't using the `luaVM` argument, there's no need to add it as a parameter